packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/index.css

```diff
@@ -31,7 +31,10 @@ limitations under the License.
 }
 
 .SpanDetail--debugValue {
+  background-color: inherit;
+  border: none;
   color: #888;
+  cursor: pointer;
```

Comment:
I think link could be a bit misleading, as it updates the current UI in addition to url. This could certainly be a separate issue that needs a bit more thought imo.